### ISSUE TITLE
 Accessibility issue

### ISSUE THREAD

**Issue by ivaldez3** [fullcalendar/fullcalendar#7845]
### Checklist

- [X] I've already searched through [existing tickets](https://github.com/fullcalendar/fullcalendar/issues)
- [X] Other people will find this feature useful

### Is this feature for a specific connector (React/Angular/etc)?

No connector in particular

### Feature Description

According to Siteimprove Accessibility Checker, it stated that the header role is missing for the calendar. 
Issue:
"Table header cell is missing a header role".
A table header cell should have the role of "columnheader" or "rowheader" as this describes its relationship to other cells in the table.

Why is this an issue?
When a screen reader user navigates a table, each data cell is announced along with the relevant column or row header. The header role provides context and meaning for the data in the table. When table header <th> elements have the role of columnheader or rowheader screen reader users can understand their relationship to each data cell.

I've attached a screenshot for the reference and a link https://www.sfcollege.edu/events.html 
![Uploading table.png…]()

**Comment by acerix** [comment]
I believe this existing ticket captures this need:
https://github.com/fullcalendar/fullcalendar/issues/6641
↑
Please visit the above issue, **give a** 👍  **to vote** ([?](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments)), press the 🔈**Subscribe** button to receive updates.
